File: app/src/main/java/com/globalwarming/earthsaver/directories/DirectoryQuestion.java
package com.globalwarming.earthsaver.directories;

public class DirectoryQuestion {

    private String id;

    private Boolean answer;

    private String category;

    private Long points;

    private String question;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public Boolean getAnswer() {
        return answer;
    }

    public void setAnswer(Boolean answer) {
        this.answer = answer;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public Long getPoints() {
        return points;
    }

    public void setPoints(Long points) {
        this.points = points;
    }

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }
}

File: app/src/main/java/com/globalwarming/earthsaver/directories/DirectoryQuestion.kt
package com.globalwarming.earthsaver.directories

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class DirectoryQuestion(
    var id: String = "",
    var answer: Boolean = false,
    var category: String = "",
    var points: Long = 0L,
    var question: String = ""
): Parcelable